picture-marker-symbols/src/main/java/com/esri/arcgisruntime/sample/picturemarkersymbols/MainActivity.java

```diff
@@ -35,168 +36,185 @@
+  MapView mMapView;
+
+  String mArcGISTempFolderPath;
+
+  String mPinBlankOrangeFilePath;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    setContentView(R.layout.activity_main);
+
+    // inflate MapView from layout
+    mMapView = (MapView) findViewById(R.id.mapView);
+
+    // create a map with the imagery basemap
+    ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+
+    // set the map to be displayed in the mapview
+    mMapView.setMap(map);
+
+    // create an initial viewpoint using an envelope (of two points, bottom left and top right)
+    Envelope envelope = new Envelope(new Point(-228835, 6550763, SpatialReferences.getWebMercator()),
+        new Point(-223560, 6552021, SpatialReferences.getWebMercator()));
+    //set viewpoint on mapview
+    mMapView.setViewpointGeometryWithPaddingAsync(envelope, 100.0);
+
+    // create a new graphics overlay and add it to the mapview
+    final GraphicsOverlay graphicOverlay = new GraphicsOverlay();
+    mMapView.getGraphicsOverlays().add(graphicOverlay);
+
+    //Add graphics using different types of picture marker symbols
+
+    //[DocRef: Name=Picture Marker Symbol URL, Category=Fundamentals, Topic=Symbols and Renderers]
+    //Create a picture marker symbol from a URL resource
+    //When using a URL, you need to call load to fetch the remote resource
+    final PictureMarkerSymbol campsiteSymbol = new PictureMarkerSymbol(
+        "http://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0/images/e82f744ebb069bb35b234b3fea46deae");
+    //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
+    //then differ across devices with different resolutions.
+    campsiteSymbol.setHeight(18);
+    campsiteSymbol.setWidth(18);
+    campsiteSymbol.loadAsync();
+    //[DocRef: END]
+    campsiteSymbol.addDoneLoadingListener(new Runnable() {
+      @Override
+      public void run() {
+        //Once the symbol has loaded, add a new graphic to the graphic overlay
+        Point campsitePoint = new Point(-223560, 6552021, SpatialReferences.getWebMercator());
+        Graphic campsiteGraphic = new Graphic(campsitePoint, campsiteSymbol);
+        graphicOverlay.getGraphics().add(campsiteGraphic);
+      }
+    });
+
+    //[DocRef: Name=Picture Marker Symbol Drawable-android, Category=Fundamentals, Topic=Symbols and Renderers]
+    //Create a picture marker symbol from an app resource
+    BitmapDrawable pinStarBlueDrawable = (BitmapDrawable) ContextCompat.getDrawable(this, R.drawable.pin_star_blue);
+    final PictureMarkerSymbol pinStarBlueSymbol = new PictureMarkerSymbol(pinStarBlueDrawable);
+    //Optionally set the size, if not set the image will be auto sized based on its size in pixels, its appearance would
```

Comment:
@willcrick You should split this comment after the comma to the next line comment.